librz/core/cmd_descs/cmd_shell.yaml

```diff
@@ -78,10 +78,26 @@ commands:
   - name: ls
     cname: cmd_shell_ls
     summary: List files and directories
+    args_str: " [-e -q -l -j] <dir/file>"
     args:
-      - name: arg
+      - name: e
+        type: RZ_CMD_ARG_TYPE_OPTION
+        flags: RZ_CMD_ARG_FLAG_OPTION
+        optional: true
+      - name: q
+        type: RZ_CMD_ARG_TYPE_OPTION
+        flags: RZ_CMD_ARG_FLAG_OPTION
+        optional: true
+      - name: l
+        type: RZ_CMD_ARG_TYPE_OPTION
+        flags: RZ_CMD_ARG_FLAG_OPTION
+        optional: true
+      - name: j
+        type: RZ_CMD_ARG_TYPE_OPTION
+        flags: RZ_CMD_ARG_FLAG_OPTION
+        optional: true
+      - name: dir/file
```

Comment:
`type: RZ_CMD_ARG_TYPE_FILE`, then it will autocomplete from the path automatically